### PR TITLE
Replace Browserify with Webpack and add DEBUG builds

### PR DESCRIPTION
**Pull request by tri170391**

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -73,6 +75,48 @@ deployment:
...
+            # Install debug streamroot-hlsjs-p2p-bundle
+            - rm -rf node_modules/streamroot-hlsjs-p2p-bundle
+            - npm install @streamroot/hlsjs-p2p-bundle-debug@beta
+            - mv node_modules/@streamroot/hlsjs-p2p-bundle-debug node_modules/streamroot-hlsjs-p2p-bundle
+            - mv node_modules/streamroot-hlsjs-p2p-bundle/hlsjs-p2p-bundle.js node_modules/streamroot-hlsjs-p2p-bundle/streamroot-hlsjs-p2p-bundle.debug.js
+
+            # Generate dist
+            - npm run build
+            - toolkit/add_banner.rb --file dist/$DIST_FILE
+                                    --deploy_env staging
```
Comment (maxbok) [inline circle.yml]: You should put `staging - debug` here.

File: circle.yml [hunk 2]
```diff
@@ -73,6 +75,48 @@ deployment:
...
+            #
+            ## End Debug builds
+            #
+
+            # Generate manifest
+            - toolkit/build_manifest.rb --name "clappr0 hls.js provider"
+                                        --version $(toolkit/current_version.rb)
+                                        --platform "web"
+                                        --artifacts ./$DIST_FILE
+                                        --debug-artifacts   "https://circleci.com/api/v1/project/streamroot/$DIST_NAME/$CIRCLE_BUILD_NUM/artifacts/0/\$CIRCLE_ARTIFACTS/$DIST_NAME.debug.js"
```
Comment (maxbok) [inline circle.yml]: Too many spaces between `--debug-artifacts` and the url!!! 😛

File: circle.yml [hunk 3]
```diff
@@ -124,6 +168,50 @@ deployment:
...
```
Comment (maxbok) [inline circle.yml]: Too many spaces between `--debug-artifacts` and the url!!! 😛

File: circle.yml [hunk 4]
```diff
@@ -124,6 +168,50 @@ deployment:
...
+            # Install debug streamroot-hlsjs-p2p-bundle
+            - rm -rf node_modules/streamroot-hlsjs-p2p-bundle
+            - npm install @streamroot/hlsjs-p2p-bundle-debug@rc
+            - mv node_modules/@streamroot/hlsjs-p2p-bundle-debug node_modules/streamroot-hlsjs-p2p-bundle
+            - mv node_modules/streamroot-hlsjs-p2p-bundle/hlsjs-p2p-bundle.js node_modules/streamroot-hlsjs-p2p-bundle/streamroot-hlsjs-p2p-bundle.debug.js
+
+            # Generate dist
+            - npm run build
+            - toolkit/add_banner.rb --file dist/$DIST_FILE
+                                    --deploy_env preprod
```
Comment (maxbok) [inline circle.yml]: You should put `preprod - debug` here.

File: circle.yml (hunk 3 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 4 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 2 above)
Comment (tri170391) [inline circle.yml]: :+1:

File: circle.yml (hunk 1 above)
Comment (tri170391) [inline circle.yml]: :+1:

**Pull request merged**